app/actors/sufia/file_set_actor.rb

```diff
@@ -0,0 +1,19 @@
+module Sufia
+  class FileSetActor < CurationConcerns::FileSetActor
+
+    def create_metadata(upload_set_id, work, file_set_params = {})
+      work ||= default_work
+      super
+    end
+
+    private
+
+      def default_work
+        GenericWork.create.tap do |w|
+          w.apply_depositor_metadata(user)
```

Comment:
what about setting upload_set_id.  I think that will be required if we want to batch apply metadata, right?